.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+## Description
+
+Please provide a short description of the change being proposed.
+
+## Type of Change
+
+- [ ] Bug Fix
+- [ ] New Feature
+- [ ] Breaking Change
+- [ ] Refactor
+- [ ] Documentation
+- [ ] Other (please describe)
+
+## Checklist
+
+- [ ] I have read the [contributing guidelines](https://github.com/wayfair-incubator/oss-template/blob/main/CONTRIBUTING.md)
```

Comment:
ought we put comments here like `<!-- TODO: update next line to point to actual contribs guidelines -->`? feels like in this template it might get lost where people should put their own content in as needed.

Reply:
Also wondering if a similar TODO line shouldn't be added to the README for greater visibility.